piker/brokers/cli.py

```diff
@@ -0,0 +1,61 @@
+"""
+Console interface to broker client/daemons.
+"""
+import json
+from functools import partial
+from importlib import import_module
+
+import click
+import trio
+
+from ..log import get_console_log, colorize_json
+
+
+def run(main, loglevel='info'):
+    log = get_console_log(loglevel)
+
+    # main sandwich
+    try:
+        return trio.run(main)
+    except Exception as err:
+        log.exception(err)
+    finally:
+        log.debug("Exiting piker")
+
+
+@click.command()
+@click.option('--broker', default='questrade', help='Broker backend to use')
+@click.option('--loglevel', '-l', default='info', help='Logging level')
+def pikerd(broker, loglevel):
+    # import broker module daemon entry point
+    brokermod = import_module('.' + broker, 'piker.brokers')
```

Comment:
Have you considered using entry points for this? http://amir.rachum.com/blog/2017/07/28/python-entry-points/

Has some good benifits if you want to make either pieces optional (entry points can be bound with optional dependencies, handling that for you) or if you want to allow other people to add their own support. I've used this for some services and it makes developing stuff out of tree easy.

Granted if its all going to be in tree, its a different story and maybe a bit more complicated than its worth.

Reply:
I'll look but for now I think since there's only one backend I'll keep everything together for the moment.